RevenueCatUI/Views/PurchaseButton.swift

```diff
@@ -63,7 +63,9 @@ struct PurchaseButton: View {
     var userInterfaceIdiom
 
     var body: some View {
-        self.button
+        Text("TESTING SNAPSHOTS")
+            .font(.largeTitle)
+            .background(.red)
```

Comment:
Should this be excluded from the PR?

Reply:
Yup, waiting for the test to run creating sample PRs (https://app.circleci.com/pipelines/github/RevenueCat/purchases-ios/14682/workflows/cc199e5f-799a-4e03-aec7-17590c569600) then I'll remove this commit :)

Reply:
Removed!